src/RadixSortLSD.chpl

```diff
@@ -159,52 +163,55 @@ module RadixSortLSD
         } // for rshift
     }//proc radixSortLSDCore
 
-    proc radixSortLSD(a:[?aD] ?t, checkSorted: bool = true): [aD] (t, int) {
+    proc radixSortLSD(a:[?aD] ?t, checkSorted: bool = true, numTasks: int = here.maxTaskPar, bitsPerDigit: int = RSLSD_bitsPerDigit): [aD] (t, int) {
```

Comment:
I would like you to pull the plan creation up higher in the call stack, in this case into the `sortMsg()` that way in the future we can choose a plan based on the input size and have less parameters.

Reply:
yeah that was always the plan, but did it this way in the interest of not changing too many things at once to keep the draft a bit easier to read. I'll go ahead and make these changes